noir-projects/noir-protocol-circuits/crates/types/src/address/aztec_address.nr

```diff
@@ -203,6 +203,14 @@ fn compute_address_from_partial_and_pub_keys() {
     assert(address.to_field() == expected_computed_address_from_partial_and_pubkeys);
 }
 
+#[test]
+fn compute_preaddress_from_partial_and_pub_keys() {
```

Comment:
this test confuses me, there is neither a partial or public key in it. Its 3 constant values hashed?

Reply:
Ye now sure what it is. But it was a ts test that expected a matching noir, might be that it should have been removed, but seems like the inner have is used in both noir and ts but just had its tet deleted in the past so just added it to silence the errors when running snapshot.

Reply:
remove the test?? its not testing what it says it is